libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigatorImpl.kt

```diff
@@ -425,6 +426,8 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
         PredictiveCacheControllerOptions(
             "",
             tileVariant,
-            MAX_NUMBER_TILES_LOAD_PARALLEL_REQUESTS
+            CacheDataDomain.MAPS,
+            MAX_NUMBER_TILES_LOAD_PARALLEL_REQUESTS,
+            0
```

Comment:
Do we want to expose this as an option? What do you think @mskurydin @kkaefer?

Reply:
cc @DmitryAk

Reply:
@Guardiola31337 It seems to me this should not be exposed right now. This could be exposed later.
@kkaefer could you please your opinion?